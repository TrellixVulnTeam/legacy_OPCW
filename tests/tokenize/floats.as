43.43626
43.43626.slap()
3246.3427237
345.4327237
(345.432).numerator
3242e329
23E2
23E-2*23E+2
42.234E32 324.3e32
