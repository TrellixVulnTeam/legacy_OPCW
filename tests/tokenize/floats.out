<stdin>:1:1-9: floating-point '43.43626'
<stdin>:2:1-9: floating-point '43.43626'
<stdin>:2:9-10: punctuator: '.'
<stdin>:2:10-14: identifier 'slap'
<stdin>:2:14-15: punctuator: '('
<stdin>:2:15-16: punctuator: ')'
<stdin>:3:1-13: floating-point '3246.3427237'
<stdin>:4:1-12: floating-point '345.4327237'
<stdin>:5:1-2: punctuator: '('
<stdin>:5:2-9: floating-point '345.432'
<stdin>:5:9-10: punctuator: ')'
<stdin>:5:10-11: punctuator: '.'
<stdin>:5:11-20: identifier 'numerator'
<stdin>:6:1-9: floating-point '3242e329'
<stdin>:7:1-5: floating-point '23E2'
<stdin>:8:1-6: floating-point '23E-2'
<stdin>:8:6-7: punctuator: '*'
<stdin>:8:7-12: floating-point '23E+2'
<stdin>:9:1-10: floating-point '42.234E32'
<stdin>:9:11-19: floating-point '324.3e32'
<stdin>:10:1-2: end
